Assets/AnkrSDK/Runtime/WalletConnect2/RpcRequests/SilentSigning/SilentSigningDisconnectRequestData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Newtonsoft.Json;
+using WalletConnectSharp.Common.Utils;
+using WalletConnectSharp.Network.Models;
+
+namespace AnkrSDK.WalletConnect2.RpcRequests.SilentSigning
+{
+	[RpcMethod("wallet_disconnectSilentSign")]
+	[RpcRequestOptions(Clock.SIX_HOURS, false, 3003)]
+	[RpcResponseOptions(Clock.SIX_HOURS, false, 3004)]
+	public class SilentSigningDisconnectRequestData : RpcRequestListDataBase
+	{
+		[Serializable]
+		public class SilentSigningDisconnectRequestParams
```

Comment:
Use a nested class when the class you are nesting is only useful to the enclosing class.

Reply:
This is the case here